rpms/generate_rpm_package.sh

```diff
@@ -45,14 +47,20 @@ clean() {
     exit ${exit_code}
 }
 
+ctrl_c() {
+    clean 1
+}
+
 build_rpm() {
     CONTAINER_NAME="$1"
     DOCKERFILE_PATH="$2"
 
+
     SOURCES_DIRECTORY="${CURRENT_PATH}/repository"
 
+
     # Download the sources
-    git clone ${SOURCE_REPOSITORY} -b $BRANCH ${SOURCES_DIRECTORY} --depth=1
+    git clone ${SOURCE_REPOSITORY} -b $BRANCH ${SOURCES_DIRECTORY} --depth=1 || clean 1
```

Comment:
`clean_and_exit` should be executed outside.

Reply:
Done in https://github.com/wazuh/wazuh-packages/pull/329/commits/2d6990de9f18cdcd7ef982c225ac0d0b83f46847.